src/main/java/com/google/api/codegen/DiscoveryContext.java

```diff
@@ -27,15 +26,15 @@
 import javax.annotation.Nullable;
 
 /**
- * A LanguageContext that provides helpers specific to the Discovery use case.
+ * A CodegenContext that provides helpers specific to the Discovery use case.
  */
-public abstract class DiscoveryContext extends LanguageContext {
+public abstract class DiscoveryContext extends CodegenContext {
 
   private final Service service;
   private final ApiaryConfig apiaryConfig;
 
   /**
-   * Constructs the abstract instance of the language provider.
+   * Constructs an abstract instance.
```

Comment:
The goal here was to turn a negative (incorrect comments) into a neutral (unhelpful comments). I slightly increase the scope of https://github.com/googleapis/toolkit/issues/62 to cover this issue.